README.md

```diff
@@ -25,11 +25,17 @@ Then bundle install:
 bundle install
 ```
 
-Then BookingSync routes need to be mounted inside you apps routes.rb:
+Then BookingSync Authorization routes need to be mounted inside you apps routes.rb:
```

Comment:
Maybe 
`Then mount BookingSync Authorization routes inside routes.rb:`

Or just change the typo to `your`